Gruntfile.coffee

```diff
@@ -110,6 +113,7 @@ module.exports = (grunt) ->
 
   # register tasks
   grunt.registerTask "build", ["clean", "coffeelint", "coffee", "concat"]
+  grunt.registerTask "watchDev", ["clean", "coffee"]
```

Comment:
Okay, I see.

But then, not sure we want to be writing code without running tests at the same time (whole essence of TDD)

but this is fine does not necessarily require any change.